src/qt/transactionrecord.cpp

```diff
@@ -330,6 +333,18 @@ bool TransactionRecord::statusUpdateNeeded(int numBlocks, int chainLockHeight) c
         || (!status.lockedByChainLocks && status.cachedChainLockHeight != chainLockHeight);
 }
 
+void TransactionRecord::updateLabel(interfaces::Wallet& wallet)
+{
+    if (IsValidDestination(txDest)) {
+        std::string name;
+        if (wallet.getAddress(txDest, &name)) {
+            label = QString::fromStdString(name);
+        } else{
```

Comment:
```suggestion
        } else {
```

This looks familiar 😄

Reply:
🙈 👍